src/js/components/dashboard/CustomAlert.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+import { RiAlertLine, RiCheckboxCircleLine, RiCloseLine, RiErrorWarningFill, RiInformationLine } from 'react-icons/ri';
+
+import NotificationType from 'consts/notificationTypes';
+
+const defaultStartIcon = {
+  [NotificationType.SUCCESS]: <RiCheckboxCircleLine />,
+  [NotificationType.INFO]: <RiInformationLine />,
+  [NotificationType.ERROR]: <RiErrorWarningFill />,
+  [NotificationType.WARNING]: <RiAlertLine />,
+};
+
+const CustomAlert = ({
+  message, classNames, customFields, id, styles, condition, handleClose,
+}) => {
+  // Show default start icon for already existing Alerts,
+  // which are not called by notification(), but Alert
+  const getStartIcon = () => {
```

Comment:
the **startIcon** naming kinda bothers me for some reason.
Since we are not dealing with multiple icons in this component I don't think it is necessary to specify the position in the prop name.
I think just an `icon` by it self should work well.

Reply:
you are right - I just thought about it like the close icon is "endIcon", but you are right it's unchangeable at the moment and we could stick to the `icon` if it's confusing for you